bridge/setu/processor/staking.go

```diff
@@ -355,7 +355,11 @@ func (sp *StakingProcessor) checkValidNonce(validatorId uint64, txnNonce uint64)
 
 	if currentNonce+1 != txnNonce {
 		sp.Logger.Error("Nonce for the given event not in order", "validatorId", validatorId, "currentNonce", currentNonce, "txnNonce", txnNonce)
-		return false, txnNonce - currentNonce, nil
+		diff := txnNonce - currentNonce
```

Comment:
Could we move logging a bit lower and also log the delay value?

Reply:
Sure

Reply:
Done